src/cmd-build

```diff
@@ -45,12 +45,13 @@ FORCE=
 FORCE_IMAGE=
 SKIP_PRUNE=0
 VERSION=
+CONTENT_SETS=/srv/redhat-coreos/content_sets.yaml
 PARENT=
 PARENT_BUILD=
 TAG=
 STRICT=
 rc=0
-options=$(getopt --options hft: --longoptions tag:,help,force,version:,parent:,parent-build:,delay-meta-merge,force-nocache,force-image,skip-prune,strict -- "$@") || rc=$?
+options=$(getopt --options hft: --longoptions tag:,help,force,version:,content_sets:,parent:,parent-build:,delay-meta-merge,force-nocache,force-image,skip-prune,strict -- "$@") || rc=$?
```

Comment:
In general, cosa uses conventions. I was thinking more that instead of passing a `--content-sets` switch, cosa just does the right thing if it detects that it's present in the configdir.